test/conftest.py

```diff
@@ -0,0 +1,31 @@
+# 3rd-party imports
+import pytest
+
+# standard imports
+import os
+import sys
+
+this_module_path = sys.modules[__name__].__file__
+
+
+@pytest.fixture(scope='module')
```

Comment:
You are not using the testing data fixture used in drtsans?

Reply:
I'm using the one in `datsans`